src/pages/installing-spark/react.mdx

```diff
@@ -2,142 +2,107 @@
-
-On sparkdesignsystem.com, we’ve documented all the Spark components with copy/paste examples for many variants and configurations you can use in your site.
-
-Let’s include a Spark Button component from the site.
-
-TODO: Instructions on accessing the Gatsby/Storybook site and getting to the React copy/paste code. Fill this section in once the site navigation is working.
+## Adding React Components
+Let’s add a component that will allow us to test that both Spark’s styles and functionality are working. We’ll use the [Loading Button](https://angular.sparkdesignsystem.com/?path=/docs/components-button--loading) component to do this. You can find code for all Spark components at [react.sparkdesignsystem.com](https://react.sparkdesignsystem.com/).
 
-## 📘 Including React Components in Your Site
-
-Add a reference to the SprkButton component at the top of App.js.
+8. Add a reference to the `SprkButton` component at the top of `App.js`.
 
 ```
 import { SprkButton } from '@sparkdesignsystem/spark-react';
 ```
 
-Copy the sample code for the Spark Button Spinner component into the return block in App.js.
-
+9. Copy the sample code into the `return` block in `App.js`.
 ```
 function App() {
-
   return (
-
     <SprkButton loading>Submit</SprkButton>
-
   );
-
 }
 ```
+You should now see a Spark Button!
 
-You should see a Spark Button with the correct Spark style applied.
-
-GIF
-
-## 🎉 You did it!
-
-You’re ready to start building with Spark! For additional help configuring and using Spark, check out some of our other guides:
-
-The Polyfills Guide will help you import JavaScript polyfills needed for Spark. Polyfills are needed for Spark to function across all modern browsers.
-
-For help importing the Spark SVG Icon Set, check out our Icon Installation Guide.
 
-Guidance on accessing and including the Rocket Sans font used by Spark can be found in our Font Installation Guide.
+<div class="sprk-u-Measure">
+  <img
+    class="sprk-u-mbm "
+    style="margin: auto; display: block;"
+    src={buttonLoaded}
+    alt="Screenshot of React App with Spark Spark Button"/>
+</div>
 
-Looking for help on theming Spark or modifying any of the default styles? Sounds like you’d be interested in our Spark Overrides Guide.
+## You did it!
+You now have a development environment set up with Spark installed and you’re ready to start  building!
 
-Working with the DatePicker component? This component requires its own configuration object which is described in the Datepicker Configuration Guide.
+For Spark support, you can [email the Spark team](mailto:sparkdesignsystem@quickenloans.com) with any questions and we’ll respond as quickly as possible.
 
+## Additional Topics
+Check out these guides for more information on setting up Spark:
 
+- The [Icon Installation Guide](/installing-spark/installing-icons) for importing the Spark SVG Icon Set.
+- The [Font Installation Guide](/installing-spark/installing-fonts) for instructions on using the Rocket Sans font.
```

Comment:
The Installing Fonts guide isn't explicitly for the Rocket Sans font, but for fonts in general. Should we update this to be more generic?

Reply:
At standup, we talked about how this might be a use case for our devs, but I also agree. Gotta talk to Dave. @yodasw16 
For now we'll keep it and change it later if needed.

Reply:
Just talked to @yodasw16, we'll keep it because 99% of the time the engineer will use the guide to install rocket sans